icinga2/service.go

```diff
@@ -3,25 +3,31 @@ package icinga2
 import (
 	"fmt"
 	"net/url"
+	"time"
 )
 
 type Service struct {
-	Name               string  `json:"name,omitempty"`
-	DisplayName        string  `json:"display_name"`
-	HostName           string  `json:"host_name"`
-	CheckCommand       string  `json:"check_command"`
-	EnableActiveChecks bool    `json:"enable_active_checks"`
-	Notes              string  `json:"notes"`
-	NotesURL           string  `json:"notes_url"`
-	ActionURL          string  `json:"action_url"`
-	Vars               Vars    `json:"vars"`
```

Comment:
We shouldn't remove the `Vars` field without providing a replacement feature which allows users to inspect all custom variables configured on a Service. If we decide to keep the field, and just document the cave-at regarding custom variables set through service templates, we may need to add `omitempty` to not overwrite custom variables set in the service template.

If we don't want to keep the field, we need a custom implementation for `GetVars()` which reads the current custom variables for the Service from the Icinga2 API directly instead.